libstorage/RocksDBStorage.cpp

```diff
@@ -54,14 +54,7 @@ Entries::Ptr RocksDBStorage::select(
         entryKey.append("_").append(key);
 
         string value;
-        auto s = m_db->Get(ReadOptions(), Slice(std::move(entryKey)), &value);
-        if (!s.ok() && !s.IsNotFound())
-        {
-            STORAGE_ROCKSDB_LOG(ERROR)
-                << LOG_DESC("Query rocksdb failed") << LOG_KV("status", s.ToString());
-
-            BOOST_THROW_EXCEPTION(StorageException(-1, "Query rocksdb exception:" + s.ToString()));
-        }
+        auto s = m_db->Get(ReadOptions(), entryKey, value);
```

Comment:
can not omit exception or move the exception handler into low-layer

Reply:
I have recovered this logic, but considerring the kind of status of rocksDB are more than LevelDB, this logic below is need to be checked. I will check it with xiaoBai